src/components/TableWithBrowserPagination/index.js

```diff
@@ -131,6 +141,8 @@ TableWithBrowserPagination.propTypes = {
     className: PropTypes.string,
     /** An object with custom style applied to the outer element. */
     style: PropTypes.object,
+    /** The variant changes the appearance of the button. Accepted variants include default and listview. */
```

Comment:
change `The variant changes the appearance of the button` to `The variant changes the appearance of the table`